properties-panel-extension/README.md

```diff
@@ -52,7 +54,7 @@ function MagicPropertiesProvider(eventBus, bpmnFactory, elementRegistry) {
     var magicTab = {
       id: 'magic',
       label: 'Magic',
-      groups: createMagicTabGroups(element, elementRegistry)
+      groups: createMagicTabGroups(element, elementRegistry, translate)
```

Comment:
Actual implementation is different:
`function createMagicTabGroups(element, translate) `